datumaro/components/extractor.py

```diff
@@ -359,6 +360,69 @@ def as_polygon(self):
     def get_area(self):
         return 0
 
+
+@attrs
+class Cuboid3d(Annotation):
```

Comment:
Isn't a cuboid inherently 3D? Seems like a redundant name.

Reply:
No, there are 2d cuboids, for example, in CVAT. https://openvinotoolkit.github.io/cvat/docs/manual/advanced/annotation-with-cuboids/creating-the-cuboid/. 3d extend them, of course, but it looks like they need another hierarchy with 3d objects.

Reply:
Okay, I guess that makes sense. 

(IMO, a cuboid is still 3D, and the thing in CVAT should be called something else, e.g. "flat cuboid", but it's subjective, so I won't insist on it.)